### PR TITLE
PyTorch SuperRes with VideoReader example

### PR DESCRIPTION
**Pull request by Kh4L**
Add full script using DALI with VideoReader operator to load frames sequences into PyTorch to train a [video superresolution model](https://arxiv.org/pdf/1707.00471.pdf).

Needed ops for the data pipeline:
- [X] VideoReader
- [x] Transpose
- [x] SequenceCrop

### CODE REVIEW

**Comment by JanuszL** [comment]
> there are some png's that I am not sure if were intended to be added to the repo (e.g. predicted.png). Were they?

It is part of the readme as I understand.

**Comment by Kh4L** [comment]
@jantonguirao @JanuszL that's correct

**Comment by JanuszL** [comment]
Please rework README.md to rst so we will have it included in our docs, and add it to our docs in appropriate place.

File: docs/examples/video/superres_pytorch/README.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+These dataloaders can be found in [dataloading/dataloaders.py](./dataloading/dataloaders.py).
+
+## Data loader performance
+
+We present performance characteristics of DALI, NVVL standalone, and a standard .png data loader
+for a number of VSRNet training scenarios. In all cases we find that DALI offers
+reduced CPU load, host memory usage, disk usage and per iteration data loading
+time.
+
+| Data loader | Input resolution | Crop size\* | Batch size | Disk space for dataset(GB) | Per iteration data time (ms) |
```
Comment (JanuszL) [inline docs/examples/video/superres_pytorch/README.md]: Table is malformed.

Comment (Kh4L) [reply docs/examples/video/superres_pytorch/README.md]: Done

Comment (Kh4L) [reply docs/examples/video/superres_pytorch/README.md]: Fixed

**Comment by JanuszL** [comment]
I think we can get rid of README.md as we have README.rst.

**Comment by Kh4L** [comment]
@JanuszL done

**Pull request merged**